app/_layout.tsx

```diff
@@ -1,19 +1,24 @@
 /* eslint-disable react-hooks/exhaustive-deps */
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import { DarkTheme, DefaultTheme, ThemeProvider } from '@react-navigation/native';
 import { useFonts } from 'expo-font';
 import { Slot, SplashScreen } from 'expo-router';
 import React, { useEffect, useLayoutEffect, useState } from 'react';
+import type { ColorSchemeName } from 'react-native';
 import { Appearance } from 'react-native';
 import { theme_colors } from '../config/theme';
 import { customFontsToLoad } from '../config/typography';
 
+type themeType = {
```

Comment:
While this works fine, I prefer to use PascalCase for types. I also think the word `Type` in the name is redundant since it is already declared right there as a type. Can we switch this to `type Theme = {...}` instead? 

I recommend watching this from the wizard himself https://youtu.be/qA65QjWCl60?t=78

Reply:
ok lemme do this right away.